src/build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 if /i "%cswinrt_echo%" == "on" @echo on
 
-set CsWinRTNet5SdkVersion=5.0.300
+set CsWinRTNet5SdkVersion=6.0.100-preview.6.21355.2
```

Comment:
How does this work?  When building for net5 projections, what happens?

Reply:
This builds both net5 and net6 projections. We could even use net5 to build both projections in cswinrt but since we build our other projects using the same .net version and .net5 does not support targeting net6 (like in test projects and WinRT.Runtime) it fails to build those projects targeting net6.